client/system_resources_test.go

```diff
@@ -0,0 +1,24 @@
+package client
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestGetSystemResources(t *testing.T) {
+	c := NewClient(GetConfigFromEnv())
+	sysResources, err := c.GetSystemResources()
+
+	if err != nil {
+		t.Fatalf("failed to get system resources with error: %v", err)
+	}
+
+	if sysResources.Uptime <= 0 {
+		t.Fatalf("expected uptime > 0, instead received '%d'", sysResources.Uptime)
+	}
+
+	version := sysResources.Version
+	if strings.Index(version, "6") != 0 && strings.Index(version, "7") != 0 {
+		t.Errorf("expected RouterOS version to start with a '7' or '6' major release, instead received '%s'", version)
+	}
+}
```

Comment:
This isn't strictly necessary since the test suite will fail miserably if the `TestMain(t *testing.M)` cannot successfully call `func(c Client) GetSystemResources()`, but I decided to add it anyway.